pages/[...slug].tsx

```diff
@@ -53,14 +51,88 @@ export default function PostPage({ toc, source, allDocs, nav, frontMatter }) {
   )
 }
 
+const settings: { [key: string]: { dir: string; tag?: string } } = {
+  'react-three-fiber': {
+    dir: 'markdown',
+    tag: 'docs-remote',
+  },
+}
+
+const cachedPages = new Map()
+
+const getPages = async (lib: keyof typeof settings) => {
+  const cached = cachedPages.get(lib)
+  if (cached) return cached
+
+  const { docs } = data.find(({ id }) => id === lib)
+  const { repo, dir, tag = 'master' } = docs
+
+  const { tree } = await fetch(
+    `https://api.github.com/repos/${repo}/git/trees/${tag}?recursive=1`
```

Comment:
curious if it's technically possible fetch from github wiki pages.
ex. https://github.com/pmndrs/valtio.wiki.git

Reply:
I'll have to give that a try. Cloning the repository (wikis can be cloned as repos) is a more robust way of doing this, although slower.

We'd have to get creative in caching by SHA or versioning and busting cache to trigger rebuilds.